source/features/repo-age.tsx

```diff
@@ -43,34 +42,28 @@ const getRepoAge = async (commitSha: string, commitsCount: number): Promise<[str
 	return [committedDate, resourcePath];
 };
 
-const getFirstCommit = cache.function(async (): Promise<[string, string] | undefined> => {
-	const commitInfo = await elementReady<HTMLAnchorElement | HTMLScriptElement>([
-		// "Repository refresh" layout
-		'.Box-header--blue .hx_avatar_stack_commit + div a[href*="/commit/"]',
-		'include-fragment[aria-label="Loading latest commit"]',
-
-		// Pre "Repository refresh" layout
-		'a.commit-tease-sha',
-		'include-fragment.commit-tease'
-	].join());
-	const commitUrl = commitInfo instanceof HTMLAnchorElement ? commitInfo.href : commitInfo!.src;
-	const commitSha = commitUrl.split('/').pop()!;
-
-	// In "Repository refresh" layout, the number of commits may not be rendered yet
-	const commitsCountElement = select('li.commits .num') ?? (await elementReady('.Box-header--blue [aria-label^="Commits on "]'))!.parentElement;
-	const commitsCount = looseParseInt(commitsCountElement!.textContent!);
-
-	// Returning undefined will make sure that it is not cached. It will check again for commits on the next load.
-	// Reference: https://github.com/fregante/webext-storage-cache/#getter
-	if (commitsCount === 0) {
```

Comment:
I just realized that this was removed. It shouldn't be

Reply:
Since until now the element could have not been ready on time so it would be undefined. Now the only case would be on a brand new repo.

Reply:
https://github.com/sindresorhus/refined-github/pull/2783/files/f976d2c0fe70d52b9adfecec5a349526d6b6a2b5#diff-9d82f672e2199b6b2b455fa80d3373f2

Reply:
> Now the only case would be on a brand new repo.

Yes. Brand new repos still exist and now I think this would throw in those cases because `repository.defaultBranchRef.target` is null

Reply:
The easiest way is to add `isEmptyRepoRoot` to `excludes`

TODO lint <-- search this next week :D

Reply:
Lol